Backpack-SwiftUI/Icons/Classes/BPKIcon.swift

```diff
@@ -15,15 +15,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import Backpack_Common
 
 public struct BPKIcon {
     public let name: String
-    let bundle: Bundle
+    let bundle: Bundle?
 
-    init (name: String, bundle: Bundle = BPKCommonBundle.iconsBundle) {
+    init(name: String, bundle: Bundle? = nil) {
```

Comment:
not using BPKCommon in the initialiser definition is the fix of the problem, as now the compiler and linker is not aware that this type exists, only at runtime this is resolved, so this makes that users of this struct dont need to import Common explicitly